### ISSUE TITLE
Operation cannot be started because the environment is not configured

### ISSUE THREAD

**Issue by kurt247** [Microsoft/PTVS#2735]
[Window Title]
Visual Studio - Python support

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[V] Show details  [Close]

[Expanded Information]
```
Build: 3.0.17114.1

System.InvalidOperationException: Operation cannot be started because the environment is not configured.
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<AbortIfNotReady>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<InstallAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtensionProvider.<InstallPackage>d__38.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtension.<InstallPackage_Executed>d__14.MoveNext()
```

**Comment by huguesv** [comment]
This looks like there's something not configured properly about your environment - could you collect the output from Tools, Python, Diagnostic Info _while your project is loaded_ and either attach it here or email to ptvshelp@microsoft.com? That will help us see what setting we are failing on.

**Comment by zooba** [comment]
This may even be as simple as pip not being installed - we do have a bug in that version where we should be displaying a message saying that you need to install pip first, but the message doesn't show and you get this error instead.

**Comment by huguesv** [comment]
Kurt, did you email us your diagnostics info?  Which interpreter are you trying to use?

**Comment by zooba** [comment]
Dup of #2495, fixed in #2502
